### PR TITLE
Getting started: finding kata

### PR DESCRIPTION
**Pull request by hobovsky**
#40 

Most of this page is reused content of original "getting started" pop-up presented currently to newbies. I reused it because I assumed that if it's still there, then it's good enough, but maybe you;d like to have it completely redone?

### CODE REVIEW

File: content/getting-started/finding-kata.md [hunk 1]
```diff
@@ -6,8 +6,42 @@ next: "/getting-started/solving-kata/"
...
-TODO kata search page
-TODO kata "numbers": rank, satisfaction, possibly others (which?) as criteria for selecting a kata to solve
--->
+Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.
+
+## Codewars trainer
+
+The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.
+
+![training routines](./img/finding-kata_01_training-routines.png)
```
Comment (kazk) [inline content/getting-started/finding-kata.md:15]: Can you take new screenshots so it shows the current UI? I'll update the popup too.

Comment (hobovsky) [reply content/getting-started/finding-kata.md:15]: I updated screenshots but it turns out I have no proper image editing software installed on my office laptop, so I am not really satisfied with the result.
Any help here would be welcome.

Comment (kazk) [reply content/getting-started/finding-kata.md:15]: We can improve it later, but here's my take:
sidebar
training-routines

We might need to retake the screenshot and make sure it's not too wide. I'll look too short on our docs.

File: content/getting-started/finding-kata.md [hunk 2]
```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
...
+
+The trainer allows you to pick your language, select your focus, preview the challenge, skip to another challenge and to start training.
```
Comment (Blind4Basics) [inline content/getting-started/finding-kata.md:17]:
```suggestion
The suggestion area allows you to preview the suggested challenge, pick your language, select your focus, then to start training or skip to another challenge.
```
to keep the order with the picture (just the preview is a bit weird to place... let's try it in first position)

File: content/getting-started/finding-kata.md [hunk 3]
```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
...
 # Finding Kata
 
-<!---
-TODO training suggestions
-TODO kata search page
-TODO kata "numbers": rank, satisfaction, possibly others (which?) as criteria for selecting a kata to solve
--->
+Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.
+
+## Codewars trainer
```
Comment (Blind4Basics) [inline content/getting-started/finding-kata.md:11]:
```suggestion
## Codewars training suggestions
```

File: content/getting-started/finding-kata.md [hunk 4]
```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
...
+
+The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.
```
Comment (Blind4Basics) [inline content/getting-started/finding-kata.md:13]:
```suggestion
The easiest way of getting started is to use the suggestions box on the dashboard to pick your next kata.
```

see #74

File: content/getting-started/finding-kata.md [hunk 5]
```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
...
+
+For newer developers, we recommend that you pick the "Fundamentals" focus track, which will randomly select easier kata for you to train on.
+
+For experienced developers looking for a challenge, you can select the "Rank Up" focus track, which will cause you to begin with easy to medium challenges and quickly advance to harder ones.
+
+If, for some reason, you do not like suggested kata, you can skip on it without any serious consequences. It will not be suggested to you by trainer anymore, but you will be always able to get back to it and solve it, for example by searching through kata library (see below).
+
+## Kata search page
+
+For those really looking for a hard challenge, we recommend you go directly to the full list of kata where you can select from over 6000 kata to train on. Remember, 1 kyū is the hardest level a kata can be.
```
Comment (Blind4Basics) [inline content/getting-started/finding-kata.md:27]:
```suggestion
For those really looking for a hard challenge or for a specific kind of rank/task/..., we recommend you go directly to the full list of kata where you can select from over 8000 kata to train on. Remember, 8 kyū is the easiest level a kata can be.
```

* #74 about passive/possessive form again (I won't mark it in the next suggestions
* changed 1 kyu to 8 kyu, this way, still up to date if 1 dan katas are created one day... x)

File: content/getting-started/finding-kata.md [hunk 6]
```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
...
+
+![sidebar](./img/finding-kata_02_sidebar.png)
+
+Details of kata search page are described in [UI documentation (TODO: insert link to documentation of kata search panel)](), but here are some hints useful for beginners:
```
Comment (Blind4Basics) [inline content/getting-started/finding-kata.md:31]:
```suggestion
Details of kata search page are described in [UI documentation (TODO: insert link to documentation of kata search panel)](), but here are some useful hints for beginners:
```

**Comment by Blind4Basics** [comment]
'forgot about the picture:

in the first picture, I'd use "Training setup" rather than "Trainer setup" (it clashes with cw's "trainer")

File: content/getting-started/finding-kata.md (hunk 3 above)
Comment (kazk) [reply content/getting-started/finding-kata.md:11]: It's called "Personal Trainer" internally. Like those in gyms.

File: content/getting-started/finding-kata.md (hunk 4 above)
Comment (kazk) [inline content/getting-started/finding-kata.md:13]:
```suggestion
The easiest way of getting started is to use the personal trainer on the dashboard to pick your next kata.
```

File: content/getting-started/finding-kata.md (hunk 3 above)
Comment (kazk) [inline content/getting-started/finding-kata.md:11]:
```suggestion
## Personal Trainer
```

File: content/getting-started/finding-kata.md (hunk 2 above)
Comment (kazk) [inline content/getting-started/finding-kata.md:17]:
```suggestion
The personal trainer suggests you a kata based on the selected language and focus area. You can either train on the suggested kata or skip to see the next suggestion.
```

**Comment by kazk** [comment]
I think it's good enough as a first draft. We'll update as we add more linked docs. Also, we should open issues for improvements on existing pages.

By the way, when merging a pull request with many commits like this one, use "Squash and merge":

image

This will keep the commit history of the master branch clean, while keeping the incremental history here. I might make this the only option.

**Pull request merged**